webcompat/static/js/lib/bugform.js

```diff
@@ -522,8 +542,13 @@ function BugForm() {
       img_url,
       ")"
     ].join("");
+    var delimeter = "[![Screenshot Description](";
```

Comment:
Do we need anything more than what's on master now? 

https://github.com/webcompat/webcompat.com/blob/master/webcompat/static/js/lib/bugform.js#L560-L562

The way things work currently:

1. user clicks submit
2. if they have an image, POST to `/upload`, get a URL back
3. stick URL in textarea, call `<form>.submit()`

If the image changes before step 1, we don't need to handle anything differently.